app/presenters/finder_presenter.rb

```diff
@@ -4,6 +4,7 @@ class FinderPresenter
 
   delegate :beta_message,
            :document_noun,
+           :context,
```

Comment:
Good point about the scoping. It's going to be used for the Policy Areas and Programmes to say what type of page it is. I'll change it now.